index.html

```diff
@@ -0,0 +1,209 @@
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care and gain insights on
+          more efficient drug therapy management.
+        </p>
+        <time datetime="2023-05-20T19:30">June 15, 2023 @ 9:00am</time>
+        <p class="venue">
+          Festival Hotel, along Amuwo Odofin, Festac town, Lagos State, Nigeria.
+        </p>
+      </section>
+    </main>
+    <!------------------------------MAIN  END------------------------------------->
+
+    <!------------------------------PROGRAMS STARTS------------------------------->
+    <section class="events">
+      <div class="container">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="events-content">
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-analysis"></i>
+              <h3 class="word">Exhibition</h3>
+            </div>
+            <p class="para para1">
+              Pharma Companies will be exhibiting their products and services.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3 class="word">Excursion</h3>
+            </div>
+            <p class="para">
+              Explore the vast Historic collections at the national Museum.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3 class="word">Workshop</h3>
+            </div>
+            <p class="para">
+              There will be Vaccination training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3 class="word">Novelty Match</h3>
+            </div>
+            <p class="para">
+              There will be a Novelty Match between the different state arm of
+              the association.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-palette"></i>
+              <h3 class="word">Group Dynamics</h3>
+            </div>
+            <p class="para">
+              There will be Group Dynamics During the conference
+            </p>
+          </div>
+          <div class="join">
+            <a href="#">Join ACPN 2023.</a>
+          </div>
+        </div>
+        <p class="show">SEE THE WHOLE PROGRAM</p>
+      </div>
+    </section>
+
+    <!------------------------------PROGRAMS  END--------------------------------->
+
+    <!------------------------------SPEAKERS  STARTS------------------------------>
+    <section class="speakers">
+      <div class="container">
+        <h2>Featured Speakers</h2>
+        <hr />
+        <div class="guest-speakers"></div>
+      </div>
```

Comment:
- [ ] Great job implementing the speakers' section, :+1: But it seems the changes you made to your code affected the layout of the speakers' image, You can see from the screenshot below that the two last images are of different sizes, please you need to look into this as all the images are meant to be of same width and height. :+1: 
SCREENSHOT:
![Screenshot from 2023-04-25 18-17-01](https://user-images.githubusercontent.com/107632780/234355752-39595bc8-1052-45fd-8dc6-6a92542532e6.png)

Reply:
Under which screen sizes?

Reply:
> Under which screen sizes?

same screen-size I pointed out in the comment above 419px to 767px . :small_airplane: :smile: